src/main/java/com/icthh/xm/tmf/ms/activation/web/rest/v4/ServiceResourceApiImpl.java

```diff
@@ -30,10 +30,9 @@
     private static final String RELATED_PARTY_ID = "relatedParty.id";
     private static final String RELATED_PARTY_REFERRED_TYPE = "relatedParty.referredType";
 
-
-    private final ServiceMapper serviceMapper;
     private final SagaService sagaService;
     private final SagaTransactionFactory sagaTransactionFactory;
+    private final ResponseEnricherServiceImpl responseEnricher;
```

Comment:
@RomanTeren why don't you use the interface here?

Reply:
my bad, I'll fix it separate PR

Reply:
https://github.com/xm-online/tmf-ms-activation/pull/94